packages/app/pages/index.tsx

```diff
@@ -47,6 +47,7 @@ const Home: NextPage = () => {
 
   const testFirebaseLogin = async (): Promise<void> => {
     const auth = getAuth();
+    console.log('got session?', session);
```

Comment:
This is probably fine for internal test client. Demoed to @SurgeCode and @microchipgnu